dockerfiles/billing-develop.dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM node:4.5
+
+RUN apt-get update
+RUN apt-get install -y git
+
+RUN mkdir /billing
+WORKDIR /billing
```

Comment:
`WORKDIR` will make the directory for you https://docs.docker.com/engine/reference/builder/#/workdir